src/stores/controller.ts

```diff
@@ -162,13 +166,32 @@ export const useControllerStore = defineStore('controller', () => {
 
     if (value === 'hidden') {
       console.warn('Window/tab hidden. Disabling joystick forwarding.')
+      if (vehicleStore.isArmed) {
+        alertStore.pushAlert(
+          new Alert(
+            AlertLevel.Warning,
+            'Warning: Vehicle armed and Cockpit minimized. Joystick inputs are stopped. Ensure vehicle safety.'
+          )
+        )
+        showElectronDesktopNotification(
+          'Vehicle armed and Cockpit minimized. Joystick inputs are stopped. Ensure vehicle safety.'
+        )
+      }
```

Comment:
In this case the alert will be disabled if the user disabled the general voice alerts or warning alerts.
We should create add a "force" parameter to the `pushAlert` method and have a dedicated switch to disable this voice alert.